src/auro-button.js

```diff
@@ -188,13 +192,16 @@ export class AuroButton extends LitElement {
         .value="${ifDefined(this.value ? this.value : undefined)}"
         @click="${() => {}}"
       >
-        ${ifDefined(this.loading ? html`<auro-loader pulse></auro-loader>` : undefined)}
+        ${ifDefined(this.loading ? html`<auro-loader pulse part="button-loader"></auro-loader>` : undefined)}
 
         <span>
           <span class="textSlot">
-            ${this.iconOnly ? undefined : html`<slot></slot>`}
+            ${this.iconOnly ? undefined : html`<slot part="button-text"></slot>`}
```

Comment:
```suggestion
            ${this.iconOnly ? undefined : html`<slot></slot>`}
```

` part="button-text"` needs to be on the span